### PR TITLE
🐛 Fix undefined method strip with extra_fields to_xml

### PR DESCRIPTION
**Pull request by josemigallas**
JIRA: [THREESCALE-8082: Expected string in extra_fields.rb](https://issues.redhat.com/browse/THREESCALE-8082)

### CODE REVIEW

**Comment by akostadinov** [comment]
If there is no way to store integers except for direct database queries, maybe just fix database and avoid this `to_s`? I wonder..

File: test/unit/extra_fields_test.rb [hunk 1]
```diff
@@ -177,10 +178,11 @@ def setup
     end
 
     # Regression: https://app.bugsnag.com/3scale-networks-sl/system/errors/61eefe2bd365260008097f85
-    test 'extra_field is an integer' do
+    test 'extra_field should not be store as an integer' do
```
Comment (akostadinov) [inline test/unit/extra_fields_test.rb:181]:
```suggestion
    test 'extra_field should not be stored as an integer' do
```

File: app/lib/fields/extra_fields.rb [hunk 2]
```diff
@@ -99,6 +99,10 @@ def encode_extra_field(value)
     case value
     when String
       encode_string_extra_field(value)
+    when Integer
+      value.to_s
+    when Array
+      value.map { |v| encode_extra_field(v) }
```
Comment (akostadinov) [inline app/lib/fields/extra_fields.rb:105]: Why do we need an array here? Should we allow arrays at all?

Comment (josemigallas) [reply app/lib/fields/extra_fields.rb:105]: Yes, extra_fields support multiple selection. See https://github.com/3scale/system/pull/2752

**Pull request merged**